contracts/GiftBucket.sol

```diff
@@ -13,6 +13,8 @@ contract GiftBucket {
 
   uint256 public expirationTime;
 
+  uint256 constant maxTxDelayInBlocks = 10;
```

Comment:
I'm happy to have it for now but we could have it as a variable in the constructor. we could open an issue for that

Reply:
ok will open an issue